### ISSUE TITLE
perceval.errors.RepositoryError is not handled

### ISSUE THREAD

**Issue by canasdiaz** [chaoss/grimoirelab-elk#552]
When the exception **perceval.errors.RepositoryError** is raised, the code in grimoirelab-elk crashes instead of handling the exception. The exception should be handled and an error message should be printed to error log.

Related issues: #550, #551, #553, #554, #555, #556

This is the traceback:
```
Error enriching ocean from git (https://github.com/openshift/openshift-httpd-cartridge-broken.git): directory '/home/bitergia/.perceval/repositories/https://github.com/openshift/openshift-httpd-cartridge-broken.git-git' for Git repository 'https://github.com/openshift/openshift-httpd-cartridge-broken.git' does not exist
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/grimoire_elk/elk.py", line 640, in enrich_backend
    enrich_count = enrich_items(ocean_backend, enrich_backend)
  File "/usr/local/lib/python3.5/dist-packages/grimoire_elk/elk.py", line 426, in enrich_items
    enrich_backend.update_items(ocean_backend, enrich_backend)
  File "/usr/local/lib/python3.5/dist-packages/grimoire_elk/enriched/git.py", line 676, in update_items
    git_repo = GitRepository(self.perceval_backend.uri, self.perceval_backend.gitpath)
  File "/usr/local/lib/python3.5/dist-packages/perceval/backends/core/git.py", line 785, in __init__
    raise RepositoryError(cause=cause)
perceval.errors.RepositoryError: directory '/home/bitergia/.perceval/repositories/https://github.com/openshift/openshift-httpd-cartridge-broken.git-git' for Git repository 'https://github.com/openshift/openshift-httpd-cartridge-broken.git' does not exist
```

**Comment by valeriocos** [comment]
PR available https://github.com/chaoss/grimoirelab-elk/pull/549. It solves also #553

**Comment by valeriocos** [comment]
Fixed with #549
